package.json

```diff
@@ -149,6 +149,12 @@
         "category": "Codacy commands",
         "icon": "$(arrow-right)"
       },
+      {
+        "command": "codacy.pr.toggleCoverageOff",
```

Comment:
is there a `toggleCoverageOn`?

Reply:
(fixed - this is shrapnel from a "cleverer" solution I wasn't clever enough to implement)